usr/src/uts/armv8/os/fakebop.c

```diff
@@ -332,6 +415,328 @@ do_bsys_alloc(bootops_t *bop, caddr_t virthint, size_t size, int align)
 	return (virthint);
 }
 
+static void
+bsetprop(int flags, char *name, int nlen, void *value, int vlen)
+{
+	uint_t size;
+	uint_t need_size;
+	bootprop_t *b;
+
+	/*
+	 * align the size to 16 byte boundary
+	 */
+	size = sizeof (bootprop_t) + nlen + 1 + vlen;
```

Comment:
You should be able to do this with `P2ALIGN`  from `sys/sysmacros.h` I think?

Reply:
Will do - this was transposition from i86pc.